onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -340,6 +340,18 @@ CUDA_VISIBLE_DEVICES=4 python3 -m models.llama.benchmark \
     --device cuda
 ```
 
+9. ONNX Runtime, FP16, convert_to_onnx, LLaMA-2 70B shard to 4 GPUs
+```
+CUDA_VISIBLE_DEVICES=4,5,6,7 bash benchmark_70b_model.sh 4 \
```

Comment:
It is better to use `CUDA_VISIBLE_DEVICES=0,1,2,3`

Reply:
why? 
We want to show we could use CUDA_VISIBLE_DEVICES to choose any GPUs we want to use for benchmark, 0,1,2,3 are default ones